staging/src/k8s.io/legacy-cloud-providers/azure/azure_instance_metadata.go

```diff
@@ -111,7 +113,7 @@ func (ims *InstanceMetadataService) getInstanceMetadata(key string) (interface{}
 
 	q := req.URL.Query()
 	q.Add("format", "json")
-	q.Add("api-version", "2017-12-01")
+	q.Add("api-version", "2019-03-11")
```

Comment:
where is this `2019-03-11` version from?

Reply:
from IMDS documentation, https://docs.microsoft.com/en-us/azure/virtual-machines/windows/instance-metadata-service